icon.browser.json

```diff
@@ -1,5 +1,11 @@
 {
     "dependencies": [
+        {
```

Comment:
Would it make more sense to just have two entry points for icons? One for background svgs and one for inline? I know we can import `@ebay/skin/base64` manually but what benefit are we getting from this flag?

Reply:
We could do that for v7 when we can introduce breaking changes, but for this minor version upgrade I'm trying to keep things backwards compatible for current apps. Or maybe I'm missing something?

Reply:
I do wonder if I could have `@ebay/skin/icon/base64` and `@ebay/skin/icon/svg` - and then `@ebay/skin/icon` just brings in both of them, thus retaining backwards compatibility.

Reply:
Yeah, then a developer could be granular, and avoid the size penalty, by pointing just to `@ebay/skin/icon/inline` without needing to have a flag. I like this line of thinking.

Reply:
@ianmcburnie I like that idea.